app/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/RecordsRepository.scala

```diff
@@ -45,14 +42,16 @@ class RecordsRepository @Inject() (
       )
     ) {
 
-  private def byEori(eori: String): Bson = Filters.equal("eori", eori)
+  private def byEori(eori: String): Bson          = Filters.equal("eori", eori)
+  private def byEoriAndActive(eori: String): Bson =
```

Comment:
Shall we add couple of unit test to cover this. We already have inactiveGoodsItemRecords in the spec which is not being used.